pandas/core/generic.py

```diff
@@ -12392,7 +12392,7 @@ def _inplace_method(self, other, op) -> Self:
         """
         warn = True
         if not PYPY and warn_copy_on_write():
-            if sys.getrefcount(self) <= 5:
+            if sys.getrefcount(self) <= 4:
```

Comment:
How did you arrive at the conclusion to change this?

Reply:
From testing in practice, but my assumption is that it is the consequence of removing the item cache, which means one reference less (so this change sounds logical in hindsight)

Reply:
Ah yeah that makes a lot of sense

Thx